.github/workflows/ci.yml

```diff
@@ -16,6 +16,9 @@ jobs:
       with:
         python-version: 3.9
 
+    - name: Tests
```

Comment:
You'll need this in a separate job, in a separate PR. To avoid copy-pasting a huge chunk of YAML, you may want to use a job matrix in this workflow.